### ISSUE TITLE
Support pipenv or poetry

### ISSUE THREAD

**Issue by aanari** [jayef0/chalice-extended-action#9]
Could I submit a PR to take additional flags to support other Python dependency management systems, rather than only using a `requirements.txt` file?

**Comment by jayef0** [comment]
Yes, of course. It would be great to support other ways of dependency management as well.
